README.md

```diff
@@ -437,6 +437,34 @@ See type ‘Address’
 |url|Where to download this file (String)||
 |checksum|MD5 checksum of this file. (String)||
 |sizeInBytes|Size of file (Long number)||
+|flags|Helpful metadata that describes the content of the file. (FileFlags)|See below|
+
+# 19. Element: FileFlags
+```javascript
+ {
+    "headToHead": true,
+    "coverSingles": true,
+    "spotUv": true,
+    "foilGold": true,
+    "foilSilver": true,
+    "foilCopper": true,
+    "foilRed": true,
+    "foilBlue": true,
+    "foilGreen": true
```

Comment:
Adding a new colour will require a new release. But then again so would doing foilColour=1 where gold=1, silver=2 etc.

Thinking about the consumption though, new keys are potentially harder to deal with than new values.